persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -1050,7 +1053,8 @@ public static String generateDatabaseConfigSyntaxTree() throws FormatterExceptio
         NodeList<ImportDeclarationNode> imports = AbstractNodeFactory.createEmptyNodeList();
         NodeList<ModuleMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
 
-        imports = imports.add(NodeParser.parseImportDeclaration(MYSQL_DRIVER_IMPORT));
+        imports = imports.add(NodeParser.parseImportDeclaration(AUTOGENERATED_FILE_COMMENT + "\n" +
```

Comment:
This should be separate from imports. From the looks of it you are using a if statement to differentiate this from other imports. 

If thats the case, shouldn't it be a different function altogether?

Reply:
This was updated in [4daf0c6](https://github.com/ballerina-platform/persist-tools/pull/79/commits/4daf0c6807a98caf258c2ef37d22f7361f78dad1). As we use the node parser to generate the import declaration with importPrefix here, we have to parse the comment with import statement

Reply:
This was updated in [6a7579c](https://github.com/ballerina-platform/persist-tools/pull/79/commits/6a7579c681ebbc4480f0ac1305335c7dd5de9571)